fec/fec/static/js/calendar.js

```diff
@@ -339,6 +348,27 @@ Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
   }
 };
 
+Calendar.prototype.handleKeypress = function(e) {
```

Comment:
It's a dumb hack to simulate clicks when enter is hit on certain calendar elements. Because they're not buttons or links, enter does nothing by default.

We could call it `simulateClick`?